app/javascript/react/components/ReviewTile.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+
+const ReviewTile = props => {
+  const rating = props.rating 
+  const body = props.body 
+    
+  return(
+    <div>
+      <li>
```

Comment:
I think the ul and li tags are reversed here. Also, do you want the rating and the body to both be list items? If so:
```
<ul>
  <li>{rating} start</li>
  <li>review: {body}</li>
</ul>
```